assets/css/nn.css

```diff
@@ -118,63 +50,43 @@ body {
         line-height: 1.846;
         color: #666666;
         background-color: #ffffff;
-}
-
-/* document header (contains title etc) */
-
-#header, #content {
-        width: 100%;
-        max-width: 62.5em;
-        margin: auto;
+        margin: 0;
 }
 
 /* headings */
-
-h2, h3, h4, h5, h6 {
-        font-family: inherit;
-        font-weight: 400;
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
         line-height: 1.1;
-        color: #ba5836;
+        color: #424242;
+        font-weight: 500;
+        font-family: 'Roboto-Medium', 'Roboto', inherit;
 }
 
 h1 {
-        font-size: 2.75em;
-        font-weight: 400;
-        font-family: 'Open Sans';
-        color: rgba(0, 0, 0, 0.7);
+        /* font-family: 'Open Sans'; */
+        font-size: 32px;
 }
 
 h2 {
-        font-size: 2.3em;
+        font-size: 20px;
 }
 
 h3 {
-        font-size: 1.6875em;
-}
-
-h4 {
-        font-size: 1.4375em;
-}
-
-h5 {
-        font-size: 1.2em;
+        font-size: 16px;
```

Comment:
TL;DR; Our website is mainly for `PC`, use adaptive `font-size` base on viewport layout in `PC` is strange, and the fixed size on mobile is just bigger. (acceptable, our customers also will focus on the font, not the picture., and so sorry we don't have a mobile design draft :(  ).

> em = desired element pixel value / parent element font-size in pixels

`em` is a computing unit, which bases its `father`. it won't be adaptive normally unless we assign his father an adaptive value. and sometimes we could not be guaranteed the size of his father's `font-size`. 

so `em` is not a good adaptive unit.

```
html {
  font-size: 16px;
}

h1 {
  font-size: 2em; // dangerous, its father is not sure, maybe not html
}
```

-------------

`rem` units are pretty like `em`, but it bases on the root element's `font-size`. and `vh/vw` base on viewport.

we could combine them and make the website adaptive.

```
html {
  font-size: 26.67vw; // base on viewport(26.67% of the viewport width), adaptive unit
  margin: 0 auto;
}
body {
  font-size: 0.14rem; // equal 26.67vw * 0.14 rem, compute unit
}
```

BUT our website mainly bases on `PC`.

For `PC`, when we magnify the viewport, the font size magnifies too. but usually, we don't like font-size changed. (`font-size` on `1024` viewport, width is not the same as `2048`, which is strange).

the best practice I think is using media queries for mobile device, and combine `px` (for PC) and `vh/vw` with `rem` (for mobile)

```css
html {
  font-size: 16px;
}

h1, h2 {
  font-size: 16px;
}

@media  screen and (max-width: 1024px) {
  html {
    font-size: 26.67vw;
  }
  body {
    font-size: 0.14rem; 
  }
}
```

and we don't design this part. so just use fixed is OK. Our customers also will focus on the font, not the picture.

Reply:
`em` could inherit its father(if his father assign or just inherit from browser).

we should not use `em` with `px`. it's dangerous.

Reply:
I was thinking em, but where you need stricter control (and don't want the size to change based on the context it appears in) rem is better.

I'm concerned that by fixing the pixels, the design is "physical" and may not scale well on all displays, and may also not honor user preferences.

While most folks coming to this site are likely coming from a regular computer, some effort was invested to make sure that the site would adapt to other conditions before, which is why the old font sizes were in em.

I guess however that these days many other sites are using physical markup in px instead of em/rem, and so folks who need to adjust have probably already done so.  I'm going to take a look again.

Reply:
Yes. Using `em` may be more friendly to some users who have a preference for font control. 

For web pages that are primarily for the `PC` side, we take control for the sake of layout certainty which ensures that the layout is consistent and simple. :)